README.md

```diff
@@ -5,6 +5,34 @@ Simple client access library for the config for the Clowder operator.
 
 Based on schema.json, the corresponding Ruby Classes are generated in types.rb.
 
+Usage
+-----
+
+To access configuration, see the following example
+
+```
+require 'app-common-ruby'
```

Comment:
The test_script.rb needs to change like this:

```
require 'app-common-ruby'

def test 
  puts "Kafka Brokers: #{LoadedConfig.kafka.brokers}" if ENV["CLOWDER_ENABLED"]
end

test
```

And use following command to test it:
```
export CLOWDER_ENABLED=true; ruby -Ilib test_script.rb
```